ccmlib/utils/sni_proxy.py

```diff
@@ -33,38 +33,46 @@ def file_or_memory(path=None, data=None):
 
 
 def create_cloud_config(ssl_dir, port, address, nodes_info, username='cassandra', password='cassandra'):
-
     def encode_base64(filename):
         return base64.b64encode(open(os.path.join(ssl_dir, filename), 'rb').read()).decode()
 
     cadata = encode_base64('ccm_node.cer')
     certificate_data = encode_base64('ccm_node.cer')
     key_data = encode_base64('ccm_node.key')
-    _, _, _, data_center = list(nodes_info)[0]  # TODO: Add ability to create cloud config for multi-DC setup
+    default_dc = "dc1"  # TODO: make default datacenter configurable
```

Comment:
at least take it from the information supplied in `node_info`, and not hard-code ?

Reply:
Fixed it.